OpenEphys.Onix/OpenEphys.Onix/ConfigureNeuropixelsV1e.cs

```diff
@@ -0,0 +1,268 @@
+
+        public const int FramesPerSuperFrame = 13;
+        public const int FramesPerRoundRobin = 12;
+        public const int AdcCount = 32;
+        public const int ChannelCount = 384;
+        public const int FrameWords = 40;
+
+        // unmanaged regiseters
+        public const uint OP_MODE = 0X00;
+        public const uint REC_MOD = 0X01;
+        public const uint CAL_MOD = 0X02;
+        public const uint TEST_CONFIG1 = 0x03;
+        public const uint TEST_CONFIG2 = 0x04;
+        public const uint TEST_CONFIG3 = 0x05;
+        public const uint TEST_CONFIG4 = 0x06;
+        public const uint TEST_CONFIG5 = 0x07;
+        public const uint STATUS = 0X08;
+        public const uint SYNC = 0X09;
+        public const uint SR_CHAIN1 = 0X0E; // Shank configuration
+        public const uint SR_CHAIN3 = 0X0C; // Odd channels
+        public const uint SR_CHAIN2 = 0X0D; // Even channels
+        public const uint SR_LENGTH2 = 0X0F;
+        public const uint SR_LENGTH1 = 0X10;
+        public const uint SOFT_RESET = 0X11;
+
+        internal class NameConverter : DeviceNameConverter
+        {
+            public NameConverter()
+                : base(typeof(NeuropixelsV1e))
+            {
+            }
+        }
+    }
+
+    [Flags]
+    public enum NeuropixelsV1CalibrationRegisterValues : uint
+    {
+        CAL_OFF = 0,
+        OSC_ACTIVE = 1 << 4, // 0 = external osc inactive, 1 = activate the external calibration oscillator
+        ADC_CAL = 1 << 5, // Enable ADC calibration
+        CH_CAL = 1 << 6, // Enable channel gain calibration
+        PIX_CAL = 1 << 7, // Enable pixel + channel gain calibration
+
+        // Useful combinations
+        OSC_ACTIVE_AND_ADC_CAL = OSC_ACTIVE | ADC_CAL,
+        OSC_ACTIVE_AND_CH_CAL = OSC_ACTIVE | CH_CAL,
+        OSC_ACTIVE_AND_PIX_CAL = OSC_ACTIVE | PIX_CAL,
+
+    };
+
+    [Flags]
+    public enum NeuropixelsV1RecordRegisterValues : uint
+    {
+        RESET_ALL = 1 << 5, // 1 = Set analog SR chains to default values
+        DIG_ENABLE = 1 << 6, // 0 = Reset the MUX, ADC, and PSB counter, 1 = Disable reset
+        CH_ENABLE = 1 << 7, // 0 = Reset channel pseudo-registers, 1 = Disable reset
+
+        // Useful combinations
+        SR_RESET = RESET_ALL | CH_ENABLE | DIG_ENABLE,
+        DIG_CH_RESET = 0,  // Yes, this is actually correct
+        ACTIVE = DIG_ENABLE | CH_ENABLE,
+    };
+
+    [Flags]
+    public enum NeuropixelsV1OperationRegisterValues : uint
+    {
+        TEST = 1 << 3, // Enable Test mode
+        DIG_TEST = 1 << 4, // Enable Digital Test mode
+        CALIBRATE = 1 << 5, // Enable calibration mode
+        RECORD = 1 << 6, // Enable recording mode
+        POWER_DOWN = 1 << 7, // Enable power down mode
+
+        // Useful combinations
+        RECORD_AND_DIG_TEST = RECORD | DIG_TEST,
+        RECORD_AND_CALIBRATE = RECORD | CALIBRATE,
+    };
+
+    public enum NeuropixelsV1ReferenceSource : byte
+    {
+        Ext = 0b001,
```

Comment:
```suggestion
        External = 0b001,
```

Maybe rename to `External` for readability and consistency with the V2 definitions?